superdesk/io/format_document_for_preview.py

```diff
@@ -0,0 +1,43 @@
+from flask import Blueprint, request, Response
+import superdesk
+from superdesk import get_resource_service
+from superdesk.publish.formatters import get_formatter
+from apps.content_types import apply_schema
+
+
+bp = Blueprint('format_document', __name__)
+
+
+def get_mime_type(formatter_qcode):
+    if(formatter_qcode == 'newsmlg2'):
```

Comment:
Any suggestions on a cleaner way of doing this? Are there any other types except XML and JSON?